oggm/core/inversion.py

```diff
@@ -126,6 +131,113 @@ def _inversion_simple(a3, a0):
     return (-a0)**cfg.ONE_FIFTH
 
 
+def shape_factor_huss(widths, heights, is_rectangular):
+    """Compute shape factor for inclusion of lateral drag
+    according to Huss and Farinotti (2012)
+
+    Discouraged to use, as this does not make any difference
+    between parabolic and rectangular beds
```

Comment:
I'm not sure to understand: wouldn't it be better to return a factor of 1 when the bed is rectangular in this case?

Reply:
I could not really find any information on the assumed bed form in the paper. Therefore I did not go into too much detail yet. But you're right, I will adapt it this way.